src/panels/PanelBase.js

```diff
@@ -97,41 +97,45 @@ export default {
   },
   methods: {
     toggle() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
-        this.isCached = true;
-      }
-    },
-    expand() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
+      if (!this.isCached && !this.localExpanded) {
+        // Let the content load before toggling localExpanded which triggers the animation,
+        // as we need to know the scrollHeight of the content for the animation to work.
         this.isCached = true;
+        this.$nextTick(() => this.localExpanded = true);
+      } else {
+        this.localExpanded = !this.localExpanded;
       }
     },
     close() {
+      this.localExpanded = false;
```

Comment:
This needs to be reset to false as well unlike before, as we want the animation to play again when re-expanding it into full form. ( this is called when collapsing a panel into its inline minimized form )